posthog/queries/session_recordings/session_recording_list.py

```diff
@@ -43,26 +43,32 @@ def __init__(self, filter: SessionRecordingsFilter, team: Team) -> None:
         LIMIT {limit}
     """
 
-    def _build_query(self):
+    def _build_query(self) -> Tuple[str, Dict]:
         distinct_id_clause = ""
-
+        distinct_id_params = {}
         if self._filter.distinct_id:
             distinct_ids = Person.objects.get(
                 team=self._team, persondistinctid__distinct_id=self._filter.distinct_id
             ).distinct_ids
-            distinct_ids_str = ",".join("'" + distinct_id + "'" for distinct_id in distinct_ids)
-            distinct_id_clause = f"AND distinct_id IN ({distinct_ids_str})"
+            distinct_id_clause = f"AND distinct_id IN %(distinct_ids)s"
+            distinct_id_params = {"distinct_ids": distinct_ids}
 
-        return self._query.format(
-            team_id=self._team.pk, distinct_id_clause=distinct_id_clause, limit=self.SESSION_RECORDINGS_DEFAULT_LIMIT
+        return (
+            self._query.format(
+                team_id=self._team.pk,
+                distinct_id_clause=distinct_id_clause,
+                limit=self.SESSION_RECORDINGS_DEFAULT_LIMIT,
+            ),
+            distinct_id_params,
         )
```

Comment:
Now I'm wondering if this could be done with a subquery. The problem is we will sometimes get a user with 20k distinct IDs, if not more, and this back-and-forth of data between postgres and django will get slow. Clickhouse complicates things obviously so this will be good enough for v1.

Reply:
Yeah, I started with the subquery approach, but then I hit the clickhouse issue. I just realized that we're duplicating the `persons` and `persons_distinct_id` tables in clickhouse, so I can move this over in the filtering PR.